backend/blueprints/spa_api/service_layers/replay/tag.py

```diff
@@ -68,7 +68,7 @@ def rename(current_name: str, new_name: str, session=None) -> 'Tag':
         # Check if name already exists
         try:
             TagWrapper.get_tag_by_name(session, get_current_user_id(), new_name)
-            raise CalculatedError(409, f"Tag with name {new_name} already exists.")
+            raise TagError(409, f"Tag with name {new_name} already exists.")
```

Comment:
TagError definition above has a 500 error while this (the error's only usage as far as I can tell) uses a 409?

Reply:
It is also a super class for tagnotfound